.github/workflows/test.yml

```
@@ -9,55 +9,46 @@ on:
     - main
 
 jobs:
-  python-django:
-    runs-on: ubuntu-latest
+  test:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```

We should simply drop Python 3.6 support. GitHub's VMs for GHA have stopped shipping unsupported Python versions. That makes things difficult.